### PR TITLE
Build tweaks

### PR DESCRIPTION
**Pull request by arkon**
- Ensure Gulp tasks run sequentially
- Lint JS files before running Gulp tasks
- Removed unused background.js file

### CODE REVIEW

**Comment by TSedlar** [comment]
Is there a certain config that gulp needs to be able to run with ES6 functions/declarations?

**Comment by arkon** [comment]
@TSedlar Whoops, it's actually the version of Node. I've removed the newer features so it works in older versions too.

File: gulpfile.js [hunk 1]
```diff
@@ -1,23 +1,25 @@
...
-var merge2 = require('merge2')
-var path = require('path')
-var runSequence = require('run-sequence')
-var source = require('vinyl-source-stream')
+'use strict'
+
+const babelify = require('babelify')
+const browserify = require('browserify')
+const del = require('del')
+const glob = require('glob')
```
Comment (TSedlar) [inline gulpfile.js]: Should this be removed since it is now unused?

Comment (arkon) [inline gulpfile.js]: Yup!

**Comment by TSedlar** [comment]
LGTM! :+1:

**Pull request merged**